ticdc/ticdc-open-api-v2.md

```diff
@@ -0,0 +1,1010 @@
+  "creator_version": "string",
+  "error": {
+    "addr": "string",
+    "code": "string",
+    "message": "string"
+  },
+  "id": "string",
+  "resolved_ts": 0,
+  "sink_uri": "string",
+  "start_ts": 0,
+  "state": "string",
+  "target_ts": 0,
+  "task_status": [
+    {
+      "capture_id": "string",
+      "table_ids": [
+        0
+      ]
+    }
+  ]
+}
+```
+
+The parameters are described as follows:
+
+| Parameter name | Description |
+|:-----------------|:---------------------------------------|
+| `admin_job_type`  | `INTEGER` type. The admin job type.                 |
+| `checkpoint_time` | `STRING` type. The formatted time of the current checkpoint for the replication task.                  |
+| `checkpoint_ts`   | `STRING` type. TSO of the current checkpoint for the replication task.    |
+| `config`          | The replication task configuration. The structure and meaning are the same as that of the `replica_config` configuration in creating the replication task.       |
+| `create_time`     | `STRING` type. The time when the replication task is created.                          |
+| `creator_version` | `STRING` type. The TiCDC version when the replication task is created.         |
+| `error`           | The replication task error.                      |
+| `id`              | `STRING` type. The replication task ID.                |
+| `resolved_ts`     | `UINT64` type. The replication task resolved ts.    |
+| `sink_uri`        | `STRING` type. The replication task sink URI.                                     |
+| `start_ts`        | `UINT64` type. The replication task start ts.                                      |
+| `state`           | `STRING` type. The replication task status. It can be `normal`, `stopped`, `error`, `failed`, and `finished`. |
+| `target_ts`       | `UINT64` type. The replication task target ts.                                    |
+| `task_status`     | The detailed status of dispatching the replication task. |
+
+The `task_status` parameters are described as follows:
+
+| Parameter name | Description |
+|:-----------------|:---------------------------------------|
+| `capture_id` | `STRING` type. The capture ID.                    |
+| `table_ids`  | `UINT64 ARRAY` type. The ID of the table being replicated on this capture. |
+
+The `error` parameters are described as follows:
+
+| Parameter name | Description |
+|:-----------------|:---------------------------------------|
+| `addr` | `STRING` type. The capture address |
+| `code` | `STRING` type. The error code          |
+| `message` | `STRING` type. The details of the error      |
+
+## Remove a replication task
+
+This API is an idempotent interface (that is, it can be applied multiple times without changing the result beyond the initial application). If the request is successful, `200 OK` is returned. The returned result only means that the server agrees to run the command but does not guarantee that the command will be run successfully.
```

Comment:
```suggestion
This API is an idempotent interface (that is, it can be applied multiple times without changing the result beyond the initial application) for removing a replication task. If the request is successful, `200 OK` is returned. The returned result only means that the server agrees to run the command but does not guarantee that the command will be run successfully.
```